Utility/UserInputHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿// can add (optional) error messages and further input (range) validation
+public static class UserInputHandler
+{
+
+    public static decimal GetValidDecimal(string output)
+    {
+        decimal? input = null;
+        while (!input.HasValue)
```

Comment:
wouldn't this be confusing to the user?

Reply:
a new version of the Class UserInputHelper.cs with more meaningful names and clear intent: https://github.com/SaifAbuR3d/ATB/blob/master/ATB.BL/Helpers/UserInputHelper.cs